ichnaea/service/locate.py

```diff
@@ -36,6 +38,14 @@
 # helper class used in searching
 Network = namedtuple('Network', ['key', 'lat', 'lon', 'range'])
 
+# Data sources for location information
+
+
+class DataSource(IntEnum):
+    MLS = 1
```

Comment:
Maybe naming this "OWN" or "OURS" would could avoid the MLS reference here. In theory ichnaea is a general purpose software and MLS just one instance of it running ... but then again the website content is full of MLS references. Nothing to do here, just a thought.